apps/web/src/components/mapping/google-maps/display-marker-info.tsx

```
@@ -27,42 +29,52 @@ const DisplayMarkerInfo = ({ location }: DisplayMarkerInfoProps) => {
+        {/* Info Card (always visible) */}
+        <div className="absolute z-30 bottom-8 left-1/2 -translate-x-1/2 w-[200px]">
+          <div className="relative overflow-hidden rounded-md bg-white shadow-xl border border-gray-200">
             {/* Image */}
-            <Image
-              src={(location?.photos && location?.photos[0]) || ""}
-              alt={location.title}
-              width={300}
-              height={300}
-              className="h-40 object-cover object-center"
-            />
+            {location?.place.photos && location?.place.photos[0] ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)